### DISCUSSION TITLE
Custom Look

### DISCUSSION THREAD

**Discussion by hpposch** [elsa-workflows/elsa-core#1489] [Q&A]
Hi all!
I was just playing around with the new version of Elsa.
I started with this example: https://elsa-workflows.github.io/elsa-core/docs/next/quickstarts/quickstarts-aspnetcore-server-dashboard-and-api-endpoints

Everything works fine and as expected.
I was now wondering if i can change the color of the topbar.
In the _Host.cshtml i can see that there is the following definition:
<link rel="stylesheet" href="/_content/Elsa.Designer.Components.Web/elsa-workflows-studio/elsa-workflows-studio.css">
This seems to be the main css file.
Is there a way that i can change it to a custom one?
I am not sure where i ahve to put my custom css file and how i can reference it.
Please help me with that.

Thanks,
Heinz Peter

**Comment by sfmskywalker** [comment]
Hi @hpposch,

Although that should definitely be possible, it may be tricky to target certain UI elements since not all key components have a unique ID or CSS classes. But you may still come a long way with a custom CSS file.

Simply create your CSS file in the wwwroot folder of your project and reference it using a `<link>` element from [_Host.cshtml](https://elsa-workflows.github.io/elsa-core/docs/next/quickstarts/quickstarts-aspnetcore-server-dashboard-and-api-endpoints#_hostcshtml).

To change the background color of the top bar, you could try the following CSS:

```css
elsa-studio-dashboard nav.elsa-bg-gray-800 {
   background-color: teal;
}
```

**Reply by hpposch** [reply]
Thank you very much, that worked for me :)
